packages/core/src/components/graphs/bubble.ts

```diff
@@ -10,10 +10,16 @@ import { scaleLinear } from "d3-scale";
 
 export class Bubble extends Scatter {
 	type = "bubble";
+	radiusLabel;
+	radiusMapsTo;
 
 	getRadiusScale(selection: Selection<any, any, any, any>) {
 		const options = this.model.getOptions();
-		const { radiusMapsTo } = options.bubble;
+		const { radiusMapsTo, radiusLabel } = options.bubble;
+		
+		// Save static variables that other components can access
+		this.radiusLabel = radiusLabel;
+		this.radiusMapsTo = radiusMapsTo;
```

Comment:
why do we need these stored in `this`?

Reply:
> why do we need these stored in `this`?
I stored them in `this` so it can be accessed in `scatter.ts`. I can't get them in `scatter.ts`

Reply:
@sophiiae this is something we should just grab off of `this.model.getOptions()` rather than storing here